lib/explorer/series.ex

```diff
@@ -2164,6 +2164,53 @@ defmodule Explorer.Series do
 
   def mean(%Series{dtype: dtype}), do: dtype_error("mean/1", dtype, [:integer, :float])
 
+  @doc """
+  Gets the most common value of the series.
+
+  ## Supported dtypes
```

Comment:
Let’s just say all except lists? Otherwise it is easy for this to go out of date!

Reply:
Yep fair 😂